controllers/territory_controller/territory_controller.py

```diff
@@ -119,7 +119,11 @@ def is_locked(self, station_code: StationCode) -> bool:
         return station_code in self._locked_territories
 
     def get_claim_count(self, station_code: StationCode) -> int:
-        return len([x for x, _, _, _ in self._log if x == station_code])
+        return len([
+            x
+            for x, y, _, _ in self._log
+            if x == station_code and y != Claimant.UNCLAIMED
```

Comment:
Feels like the positive test here would be clearer, especially given what the but was:
```suggestion
            if x == station_code and y == Claimant.CLAIMED
```

Reply:
Except that `Claimant.CLAIMED` isn't a thing...

Reply:
5f124e202bcf7c6f8cdce7d3668beccc4651859d